src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -456,4 +469,13 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
 .tool-tip-text{
    border-bottom: 1px dashed $gray6;
 }
+
+#create-special-resolution{
+.invalid-section{
+  ::v-deep label.v-label.error--text.theme--light ,label {
+    color: $BCgovInputError !important;
+  }
+
+  }
```

Comment:
Did you try something simpler than this, like `#create-special-resolution .invalid-section .v-label { ... }`?

Reply:
I think this is a better approach, since we are using scss we can group css